src/bank/BankActions.java

```diff
@@ -0,0 +1,44 @@
+package bank;
+
+import currency.Currency;
+
+import java.time.LocalDate;
```

Comment:
usually try to remove unused imports

Reply:
Am facut un nou push in care am remediat aceasta situatie.